tests/unit/dataactbroker/test_submission_handler.py

```diff
@@ -56,7 +56,7 @@ def test_get_submission_metadata_quarterly_dabs_cgac(database):
         'agency_name': cgac.agency_name,
         'number_of_errors': 40,
         'number_of_warnings': 200,
-        'number_of_rows': 10,
+        'number_of_rows': 8,
```

Comment:
Great way to prove that multiple files will be accounted for multiple header lines.